src/app/studio/[surveyId]/page.tsx

```diff
@@ -1,34 +1,22 @@
-import { Button } from "@/components/ui/button";
 import CreateQuestionDialog from "@/components/forms/create-question-form";
-import {
-  Card,
-  CardContent,
-  CardDescription,
-  CardFooter,
-  CardHeader,
-  CardTitle,
-} from "@/components/ui/card";
+import { authOptions } from "@/pages/api/auth/[...nextauth]";
+import { getServerSession } from "next-auth";
+import { notFound, redirect } from "next/navigation";
 
-export default function Studio({ params }: { params: { surveyId: string } }) {
-  return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <Button>Hier</Button>
+export default async function Studio({
+  params,
+}: {
+  params: { surveyId: string };
+}) {
+  const session = await getServerSession(authOptions);
 
-      <CreateQuestionDialog surveyId={params.surveyId} />
-      <Card className="min-w-[20rem]">
-        <CardHeader>
-          <CardTitle>Card Title</CardTitle>
-          <CardDescription>Card Description</CardDescription>
-        </CardHeader>
-        <CardContent>
-          <p>Card Content</p>
-        </CardContent>
-        <CardFooter>
-          <p>Card Footer</p>
-          <Button>Button</Button>
-          <Button>Button</Button>
-        </CardFooter>
-      </Card>
-    </main>
-  );
+  if (!session) {
+    return redirect("/api/auth/signin") || notFound();
```

Comment:
I think just returning the redirect is sufficient. `|| notFound()` does not make sense here as it will never be reached